src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1245,21 +1245,24 @@
 				var ui = this._ui,
 					state = this._state,
 					selectedElement = state.items[selectedIndex].element,
-					marqueeDiv,
-					widget;
+					marqueeWidget = state.items[selectedIndex].marqueeWidget,
```

Comment:
Instead of cache the marquee widget, please use `ns.engine.getBinding()` to get widget instance.
eg:
```
marqueeWidget = ns.engine.getBinding(selectedElement.querySelector(".ui-arc-listview-text-content"));
```

Reply:
Ok. Thanks.